targets/TARGET_NORDIC/TARGET_NRF5x/TARGET_SDK_11/softdevice/common/softdevice_handler/softdevice_handler.c

```diff
@@ -490,22 +484,18 @@ uint32_t softdevice_enable(ble_enable_params_t * p_ble_enable_params)
                     app_ram_base);
     err_code = sd_ble_enable(p_ble_enable_params, &app_ram_base);
 
-#if defined(NRF_LOG_USES_RTT) && NRF_LOG_USES_RTT == 1
+    // MODIFIED the rest of this function to match logging implementation in SDK13
     if (app_ram_base != ram_start)
     {
-        uint32_t app_ram_size= ram_end_address_get();
-        SD_HANDLER_LOG("sd_ble_enable: app_ram_base should be adjusted to 0x%x\r\n",
-                  app_ram_base);
-        app_ram_size -= app_ram_base;
-        SD_HANDLER_LOG("ram size should be adjusted to 0x%x \r\n",
-                  app_ram_size);
+        SD_HANDLER_LOG("sd_ble_enable: RAM start should be adjusted to 0x%x\r\n",
+                app_ram_base);
+        SD_HANDLER_LOG("RAM size should be adjusted to 0x%x \r\n",
+                ram_end_address_get() - app_ram_base);
     }
     else if (err_code != NRF_SUCCESS)
     {
         SD_HANDLER_LOG("sd_ble_enable: error 0x%x\r\n", err_code);
-        while(1);
```

Comment:
I don't think we should remove this

Reply:
Newer versions of the SDK don't hang here, but I guess it would also be more useful to redirect this to mbed_error()... though that would being in even more changes and dependencies so perhaps leaving it in place would be ok.